src/base/rss/rss_session.h

```diff
@@ -161,6 +164,7 @@ namespace RSS
 
         CachedSettingValue<bool> m_storeProcessingEnabled;
         CachedSettingValue<int> m_storeRefreshInterval;
+        CachedSettingValue<int> m_storeFetchDelay;
```

Comment:
I would suggest using a proper data type:
```suggestion
        CachedSettingValue<std::chrono::seconds> m_storeFetchDelay;
```

Reply:
It looks great, more readable and safer, didn't know std had chrono.
I will add those suggestions.